dcrutil/tx.go

```diff
@@ -79,6 +79,16 @@ func (t *Tx) SetTree(tree int8) {
 	t.txTree = tree
 }
 
+// Version returns the transaction version.
+func (t *Tx) Version() uint16 {
+	return t.msgTx.Version
+}
+
+// SetVersion sets the version of the transaction within a block.
```

Comment:
This function is only setting the transaction version, `within a block` in the description should be removed.